docs/README.md

```diff
@@ -16,7 +16,250 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+You can follow the steps below to install manually or you can run [this script](multicluster/setup-multi-primary.sh) which will setup a local environment for you with kind. Before running the setup script, you must install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) and [cloud-provider-kind](https://kind.sigs.k8s.io/docs/user/loadbalancer/#installing-cloud-provider-kind) then ensure the `cloud-provider-kind` binary is running in the background.
+
+1. Setup env vars
+
+    ```sh
+    export CTX_CLUSTER1=<cluster1-ctx>
+    export CTX_CLUSTER2=<cluster2-ctx>
+    export ISTIOCTL_DIR=<path-to-istioctl-dir>
+    ```
+
+2. Create `istio-system` namespace on each cluster
+
+    ```sh
+    kubectl get ns istio-system --context "${CTX_CLUSTER1}" || kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+    ```
+
+    ```sh
+    kubectl get ns istio-system --context "${CTX_CLUSTER2}" || kubectl create namespace istio-system --context "${CTX_CLUSTER2}"
+    ```
+
+3. Create shared trust and add intermediate CAs to each cluster.
+
+    If you already have a [shared trust](https://istio.io/latest/docs/setup/install/multicluster/before-you-begin/#configure-trust) for each cluster you can skip this. Otherwise, you can use the instructions below to create a shared trust and push the intermediate CAs into your clusters.
+
+    Create a self signed root and intermediate CAs.
+    ```sh
+    mkdir -p certs
+    pushd certs
+    make -f "${ISTIOCTL_DIR}/tools/certs/Makefile.selfsigned.mk" root-ca
+    make -f "${ISTIOCTL_DIR}/tools/certs/Makefile.selfsigned.mk" east-cacerts
+    make -f "${ISTIOCTL_DIR}/tools/certs/Makefile.selfsigned.mk" west-cacerts
+    popd
+    ```
+
+    Push the intermediate CAs to each cluster.
+    ```sh
+    kubectl --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+    kubectl get secret --context "${CTX_CLUSTER1}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+          --from-file=certs/east/ca-cert.pem \
+          --from-file=certs/east/ca-key.pem \
+          --from-file=certs/east/root-cert.pem \
+          --from-file=certs/east/cert-chain.pem
+    kubectl --context "${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+    kubectl get secret --context "${CTX_CLUSTER2}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER2}" \
+          --from-file=certs/west/ca-cert.pem \
+          --from-file=certs/west/ca-key.pem \
+          --from-file=certs/west/root-cert.pem \
+          --from-file=certs/west/cert-chain.pem
+    ```
+
+4. Create Sail CR on east
+
+    ```sh
+    kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
```

Comment:
Should we create this in the `istio-system` namespace as otherwise it will be created in `default` namespace.

Reply:
Namespace where the istiod gets put is set in `spec.namespace` rather than `.metadata.namespace` because `Istio` is a cluster scoped resource.